cli/ceph/config.py

```diff
@@ -17,3 +17,28 @@ def set(self, key, value):
         """
         cmd = f"{self.base_cmd} set {key} {value}"
         return self.execute(sudo=True, cmd=cmd)
+
+    def set_osd_memory_autotune(self, value):
```

Comment:
`set_osd_memory_autotune` and `set_host_osd_memory_target` can be merged with `set` method